mods/d2k/mod.yaml

```diff
@@ -128,7 +129,7 @@ ContentInstaller:
-	CopyFilesFromCD: music/ambush.aud, music/arakatak.aud, music/atregain.aud, music/entordos.aud, music/fightpwr.aud, music/fremen.aud, music/hark_bat.aud, music/landsand.aud, music/options.aud, music/plotting.aud, music/risehark.aud, music/robotix.aud, music/score.aud, music/soldappr.aud, music/spicesct.aud, music/undercon.aud, music/waitgame.aud
+	CopyFilesFromCD: music/ambush.aud, music/arakatak.aud, music/atregain.aud, music/entordos.aud, music/fightpwr.aud, music/fremen.aud, music/hark_bat.aud, music/landsand.aud, music/options.aud, music/plotting.aud, music/risehark.aud, music/robotix.aud, music/score.aud, music/soldappr.aud, music/spicesct.aud, music/undercon.aud, music/waitgame.aud, movies/A_BR01_E.VQA, movies/A_BR02_E.VQA, movies/A_BR03_E.VQA, movies/A_BR04_E.VQA, movies/A_BR05_E.VQA, movies/A_BR06_E.VQA, movies/A_BR07_E.VQA, movies/A_BR08_E.VQA, movies/A_BR09_E.VQA, movies/A_FINL_E.VQA, movies/A_LOSE_E.VQA, movies/A_MNTG_E.VQA
```

Comment:
Just install them all at once. You won't have a second chance as they are not part of the TestFiles and we don't have a download/install or even update logic yet.